docs/docs/20-quickstart.mdx

```diff
@@ -137,9 +139,17 @@ helm install kargo \
   --version 0.1.0-rc.21 \
   --namespace kargo \
   --create-namespace \
+  --set api.service.type=NodePort \
+  --set api.service.nodePort=30084 \
+  --set 'api.adminAccount.password=admin' \
   --wait
 ```
 
+The Kargo dashboard will be accessible at
+[localhost:8084](http://localhost:8084).
+
+The Admin password is `admin`.
```

Comment:
I don't care too much one way or another, but dummy SSO should work right out of the box, which may be even easier than this.

Either way, I'm good with it.

Reply:
I prefer the consistency of both UIs having the admin user and password.

But for my curiosity, how would I enable the dummy SSO?

Reply:
I misspoke... it is not enabled by default, so please disregard. 🤦‍♂️